### PR TITLE
Update HackerWeb in showcase

### PR DESCRIPTION
**Pull request by cheeaun**
Tiny little update: adding another blog post link on [how I build the Android app](http://cheeaun.com/blog/2016/05/building-hackerweb-android/).

### CODE REVIEW

**Comment by facebook-github-bot-6** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -443,6 +443,7 @@ var apps = [
     author: 'Lim Chee Aun',
     blogs: [
       'http://cheeaun.com/blog/2016/03/building-hackerweb-ios/',
+      'http://cheeaun.com/blog/2016/05/building-hackerweb-android/'
```
Comment (dsibiski) [inline website/src/react-native/showcase.js]: Please add a trailing comma here, thanks! 👍

**Comment by facebook-github-bot-4** [comment]
@cheeaun updated the pull request.

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-4** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1758067787764161/int_phab) to review.

**Pull request closed without merging**